common/views/components/Pagination/Pagination.js

```diff
@@ -2,8 +2,9 @@
 
 import {font, spacing} from '../../../utils/classnames';
 import Icon from '../Icon/Icon';
+import Link from 'next/link';
```

Comment:
Ah, yeah – here.

Reply:
Exactly.

Reply:
Would you mind updating this one to NextLink while you're at it?